app/controllers/reviews_controller.rb

```diff
@@ -0,0 +1,35 @@
+class ReviewsController < ApplicationController
+  before_action :find_review, only: %i[show]
```

Comment:
before_action :set_job + define set_job in private
def set_job
 @job = Job.find(params[:job_id])
end